pints/tests/test_toy_stochastic_degradation_model.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+#
+# Tests if the stochastic degradation (toy) model works.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import unittest
+import numpy as np
+import pints
+import pints.toy
+
+
+class TestStochasticDegradation(unittest.TestCase):
+    """
+    Tests if the stochastic degradation (toy) model works.
+    """
+
+    def test_start_with_zero(self):
+        # Test the special case where the initial concentration is zero
+        from pints.toy import StochasticDegradationModel
+        model = StochasticDegradationModel(0)
+        times = [0, 1, 2, 100, 1000]
+        parameters = [0.1]
+        values = model.simulate(parameters, times)
+        self.assertEqual(len(values), len(times))
+        self.assertTrue(np.all(values == np.zeros(5)))
+
+    def test_start_with_twenty(self):
+        # Run small simulation
+        model = pints.toy.StochasticDegradationModel(20)
+        times = [0, 1, 2, 100, 1000]
+        parameters = [0.1]
+        values = model.simulate(parameters, times)
+        self.assertEqual(len(values), len(times))
+        self.assertEqual(values[0], 20)
+        self.assertEqual(values[-1], 0)
+
```

Comment:
Maybe add a check like ```self.assertTrue(values[1:] <= values[:-1]``` to check the values are monotonically decreasing

Reply:
Done